ops/helmfiles/config/fungible.yaml

```diff
@@ -24,6 +24,26 @@ collator:
   enableServiceMonitorCollectors: true
   wsHealthExporter:
     enabled: false
+  collator01:
+    # Identity ID: 12D3KooWAfm5cK7DuUa2zh9SrSoHr2fsXXYbLALWXs5ob9mTDKHJ
+    customNodeKey: '5ad94653edf592f59d1b54960cda1334596cc3dcab8dd96adb334d3e22e912ef'
```

Comment:
Don't we want default to be different from fungible in case we deploy accidentally without setting the config correctly causing a conflict...